terraform/modules/aws_ecs_service/main.tf

```diff
@@ -1,10 +1,10 @@
 ########################################################################
 # Data Bits
 ########################################################################
-data "aws_ecr_repository" "service" {
-  count = var.use_ecr ? 1 : 0
-  name  = var.ecr_repository_name
-}
+# data "aws_ecr_repository" "service" {
+#   count = var.use_ecr ? 1 : 0
```

Comment:
This is a good change IMHO 🥳  But let's remove the `use_ecs` in the `variables.tf` also then delete this block and keep your other change in this module. Nice work

Reply:
We still need use `use_ecr` in other parts of main. For example: https://github.com/NASA-IMPACT/veda-features-api/blob/c3109acf7f3b92b364e8965be7bf19435e865c07/terraform/modules/aws_ecs_service/main.tf#L53. So should we still keep it?

Reply:
Doh, I didn't check that. I don't think we need it. What we're basically doing here is saying, "let's just create the role with all the permissions we think it needs for ECR too" instead of having the user tell us at function call time whether they need it or not. I think that's fine.

But are you comfortable changing all those things? If not just abort all changes and I can handle it in the future